common/rya.api/src/main/java/org/apache/rya/api/client/CreatePCJ.java

```diff
@@ -44,7 +44,7 @@
      * Application.
      *
      */
-    public static enum ExportStrategy{RYA, KAFKA, NO_OP_EXPORT};
+    public static enum ExportStrategy{RYA_EXPORT, KAFKA_EXPORT, NO_OP_EXPORT, PERIODIC_EXPORT};
```

Comment:
I feel like the `_EXPORT` suffix is a bit redundant given that the enum is `ExportStrategy`.  Also, the `NO_OP_EXPORT` could probably just be renamed to `NONE`.

Reply:
Hmm, since you can have multiple export strategies, I don't see why we can't just remove the `NO_OP_EXPORT` enum and the `NoOpExporter` class entirely.  If you get an empty set of strategies, that means you never have to obtain an exporter that doesn't do anything.

Reply:
Done.  NO_OP_EXPORT deleted from existence.  I will sincerely miss all of the super important no-opting that it was doing.